hub/hub.go

```diff
@@ -0,0 +1,133 @@
+// Hub is a package that describes the Hubs and clients functionalities and
+// structure for the "web-chat-go" usage.
+package hub
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+)
+
+var wg = &sync.WaitGroup{}
```

Comment:
`wg` is normally a local scope var name. globals should have the context in the name